gradle/version.gradle

```diff
@@ -1,6 +1,6 @@
 import java.util.regex.Pattern
 
-version = "2.0.22"//detectSemVersion()
+version = "2.1.0"//detectSemVersion()
```

Comment:
should we bump majpr version of commons to 3.0.0 ? as it is quite big changes.
@ishkurko @major13ua @sergeysenja1992  what do you think?

Reply:
@amedvedchuk Propose to use 2.1.0 to align with spring boot.

Reply:
@ishkurko we should not align to Spring boot version as it is separate component. I think we can not and should not put such constraint to xm-commons module